includes/admin/plugins.php

```diff
@@ -325,6 +324,20 @@ function perflab_render_plugin_card( array $plugin_data ): void {
 			'<button type="button" class="button button-disabled" disabled="disabled">%s</button>',
 			esc_html( _x( 'Active', 'plugin', 'default' ) )
 		);
+		$all_plugins    = array_keys( get_plugins() );
+		$plugin_slug    = $plugin_data['slug'];
+		$plugin_path    = '';
+		foreach ( $all_plugins as $path ) {
```

Comment:
Might as well just do this instead of adding the `$all_plugins` variable.

```suggestion
		foreach ( array_keys( get_plugins() ) as $file ) {
```